app/components/Settings/Preview/code-editor.jsx

```diff
@@ -25,6 +25,7 @@ export default class CodeEditor extends React.Component {
 
         dialect: PropTypes.string,
         runQuery: PropTypes.func,
+        scheduleQuery: PropTypes.func,
```

Comment:
see [this comment](https://github.com/plotly/falcon-sql-client/pull/476/files#r200124964)